From: Fabricio Silva <hi@fabricio.dev>
Date: Mon, 7 Aug 2023 09:29:47 +0100
Subject: [PATCH] Disable GitVersion

GitVersion is a dependency that needs git history to return the current version.
APKBUILD does not provide git history. Disable it and replace version in prepare() step.

---
 src/Recyclarr.Cli/Program.cs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/Recyclarr.Cli/Program.cs b/src/Recyclarr.Cli/Program.cs
index 6befa96..d98d643 100644
--- a/src/Recyclarr.Cli/Program.cs
+++ b/src/Recyclarr.Cli/Program.cs
@@ -38,6 +38,7 @@ internal static class Program
             config.Settings.StrictParsing = true;

             config.SetApplicationName("recyclarr");
+            config.SetApplicationVersion("{Version}");

             var interceptor = new CliInterceptor(logLevelSwitch, appDataPathProvider);
             interceptor.OnIntercepted.Subscribe(_ => OnAppInitialized());
@@ -78,7 +79,7 @@ internal static class Program
         }

         _log = _scope.Resolve<ILogger>();
-        _log.Debug("Recyclarr Version: {Version}", GitVersionInformation.InformationalVersion);
+        _log.Debug("Recyclarr Version: {Version}");

         _tasks = _scope.Resolve<IOrderedEnumerable<IBaseCommandSetupTask>>().ToArray();
         _tasks.ForEach(x => x.OnStart());
